packages/css/src/transform.ts

```diff
@@ -49,6 +51,7 @@ export const transformCss = (
         classNameCompressionMap: opts.classNameCompressionMap,
```

Comment:
There are quite a lot of references of `classNameCompression` in the code base. Since we don't support this feature, we should clean them up. 

I am happy to do it after this PR.

Reply:
lets not clean up `classNameCompression` from elsewhere in the codebase yet, as we might use it for future performance projects